testsuite/gnat2goto/tests/access-to-object/main.adb

```diff
@@ -0,0 +1,16 @@
+procedure Main is
+   type Day_Of_Month is range 1 .. 31;
+   type Day_Of_Month_Access is access all Day_Of_Month;
+
+   -- funny case where the subtype-indication has subtype-mark
+   type C_String_Ptr is access String (1 .. Positive'Last);
```

Comment:
So this is defining a pointer to string type but not actually using it??

Reply:
`type Restricted_Day_Of_Month_Access is access all Day_Of_Month (range 10..15);`

Reply:
Changed.